galette/templates/default/page.html.twig

```diff
@@ -95,5 +95,12 @@
         </a>
         {% include "elements/scripts.html.twig" %}
         {% block javascripts %}{% endblock %}
+        {# If some additional scripts should be added from plugins, we load the relevant template file
+           We have to use a template file, so Twig will do its work (like replacing variables). #}
+        {% if plugin_scripts|length != 0 %}
+            {% for mid, script in plugin_scripts %}
+                {% include script with {'module_id': mid} %}
+            {% endfor %}
+        {% endif %}
```

Comment:
Should not that be part of the "elements/scripts.html.twig"? Would prevent to add you code twice :)

Reply:
The plugin I'm working on needs to be able to "control" code set in the javascript block of the pages templates. But I couldn't just move the "elements/scripts.html.twig" after the javascript block in the templates as this could break the code defined in that block.

That's why I ended with this solution.

FYI, one goal of my module is to provide a Consent Management Platform.

Reply:
My idea was to move the javascript block and your new code at the end of the `scripts.html.twig` file; not only your new code (sorry if I was not clear enough).

Reply:
As `scripts.html.twig` is an include, moving the javascripts block this way would break inheritance in templates that extend `page.html.twig` and `public_page.html.twig` :confused:

Reply:
Oh.. OK, I did not think this would break. OK for me then!